code/game/objects/structures/ai_core.dm

```diff
@@ -259,14 +300,38 @@
 				if(istype(P, /obj/item/aicard))
 					return //handled by /obj/structure/ai_core/transfer_ai()
 
-				if(P.tool_behaviour == TOOL_SCREWDRIVER)
+				if(P.tool_behaviour == TOOL_WIRECUTTER)
 					P.play_tool_sound(src)
-					to_chat(user, span_notice("You disconnect the monitor."))
+					balloon_alert(user, "disconnected the monitor")
 					state = GLASS_CORE
 					update_appearance()
 					return
 	return ..()
 
+/obj/structure/ai_core/proc/ai_structure_to_mob()
+	var/mob/living/brain/B = brain.brainmob
+	if(!B.mind)
+		return FALSE
+	B.mind?.remove_antags_for_borging()
+
+	var/mob/living/silicon/ai/A = null
+
+	if(brain.overrides_aicore_laws)
+		A = new /mob/living/silicon/ai(loc, brain.laws, B)
+		brain.laws = null //Brain's law datum is being donated, so we need the brain to let it go or the GC will eat it
+	else
+		A = new /mob/living/silicon/ai(loc, laws, B)
+		laws = null //we're giving the new AI this datum, so let's not delete it when we qdel(src) 5 lines from now
+
+	if(brain.force_replace_ai_name)
+		A.fully_replace_character_name(A.name, brain.replacement_ai_name())
+	if(brain.braintype == "Android")
+		A.posibrain_core = TRUE
+	SSblackbox.record_feedback("amount", "ais_created", 1)
+	deadchat_broadcast(" has been brought online at <b>[get_area_name(A, TRUE)]</b>.", span_name("[A]"), follow_target=A, message_type=DEADCHAT_ANNOUNCEMENT)
```

Comment:
Named parameter for the `TRUE` in `get_area_name`. Also this looks like a copy post.

Reply:
@Mothblocks This is the same code that would execute currently if you used a screwdriver on a `state == GLASS_CORE` AI, i just moved it to a separate proc